### ISSUE TITLE
[Request] Add a wiki

### ISSUE THREAD

**Issue by Poltergeisen** [khellang/Scrutor#107]
I am a fairly inexperienced developer who might benefit from a wiki page explaining when you'd use this library and what it solves that the base .net framework does not.

I was under the impression that .Net core had assembly scanning already. Does this just make it more user-friendly or is it something else?

**Comment by khellang** [comment]
> I was under the impression that .Net core had assembly scanning already. Does this just make it more user-friendly or is it something else?

No, the framework itself does not do assembly scanning (that's why I wrote this library 😉) , but it contains the primitives to do it (reflection).

**Comment by Poltergeisen** [comment]
My mistake. I had gotten the assembly scanning mixed up with a different library! Sorry about that.
